usaspending_api/etl/es_transaction_template.json

```diff
@@ -1,10 +1,10 @@
 {
-  "index_patterns": ["*"],
+  "index_patterns": ["*transactions*"],
```

Comment:
This will be overwritten when the template is loaded into an ES cluster using the `es_configure` management command

Reply:
ok